src/markdown.rs

```diff
@@ -23,18 +23,24 @@ pub fn prepare_markdown(text: &str) -> String {
     return buf;
 }
 
-pub fn make_skin() -> termimad::MadSkin {
+fn make_skin() -> termimad::MadSkin {
+    if !atty::is(atty::Stream::Stdout) {
+        return termimad::MadSkin::no_style();
+    }
+
     use crossterm::style::{Color, Attribute};
```

Comment:
Please move use at the top of the block. And keep a line separating `use` from code. Or just move it to the module level. We usually import at function level either enums like `Color::*` or platform-specific stuff, or ocasionally if we need to override something.

Reply:
Sure, moved to the top of the block. I'm also fine to move it to the top of the module, my motivation to keep it in the function was that those names are only used in it, and the function is like 10% of the file. But again, I'm all for consistent code style so let me know if you want to move it to the module level.